src/components/sidebar/sidebar.js

```diff
@@ -55,21 +62,26 @@ export const BSidebar = /*#__PURE__*/ Vue.extend({
       // default: null
     },
     closeLabel: {
-      // `aria-label` for close button. Defaults to 'Close'
+      // `aria-label` for close button
+      // Defaults to 'Close'
       type: String
-      // defaut: undefined
+      // default: undefined
     },
     tag: {
       type: String,
-      default: 'div'
+      default: () => getComponentConfig(NAME, 'tag')
     },
     headerClass: {
       type: [String, Array, Object]
       // default: null
     },
+    contentClass: {
+      type: [String, Array, Object]
+      // default: null
+    },
     noSlide: {
       type: Boolean,
-      default: false
+      default: () => getComponentConfig(NAME, 'noSlide')

```

Comment:
We don't allow boolean only props to be in the config.

So this should be hard coded to `false`

Reply:
fixed